src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/themes/AceThemes.java

```diff
@@ -74,6 +78,33 @@ private void applyTheme(Document document, final AceTheme theme)
       currentStyleEl.setRel("stylesheet");
       currentStyleEl.setId(linkId_);
       currentStyleEl.setHref(themeUrl.toString());
+      
+      // In server mode, augment the theme with a font if we have one
+      LinkElement fontEl = null;
```

Comment:
Might as well move this down to line 89 where it is first used (and only used inside that block as far as I can see).